zhaquirks/schneiderelectric/clusters/__init__.py

```diff
@@ -0,0 +1,172 @@
+            ),
+        }
+    )
+
+
+class SEOnOff(SEManufCluster, OnOff):
+    attributes = OnOff.attributes.copy()
+    attributes.update(
+        {
+            0xE000: ("unknown_attribute_57344", t.uint16_t),
+            0xE001: ("unknown_attribute_57345", t.uint32_t),
+            0xE002: ("unknown_attribute_57346", t.bitmap8),
+            0xE003: ("unknown_attribute_57347", t.uint32_t),
+        }
+    )
+
+
+class SEWindowCovering(SEManufCluster, WindowCovering):
+    """Manufacturer Specific Cluster of cover device."""
+
+    attributes: dict[int, ZCLAttributeDef] = WindowCovering.attributes.copy()
+
+    attributes.update(
+        {
+            0xFFFD: ("unknown_attribute_65533", t.uint16_t),
+            0xE000: ("lift_duration", t.uint16_t),
+            0xE010: ("unknown_attribute_57360", t.bitmap8),
+            0xE012: ("unknown_attribute_57362", t.uint16_t),
+            0xE013: ("unknown_attribute_57363", t.bitmap8),
+            0xE014: ("unknown_attribute_57364", t.uint16_t),
+            0xE015: ("unknown_attribute_57365", t.uint16_t),
+            0xE016: ("unknown_attribute_57366", t.uint16_t),
+            0xE017: ("unknown_attribute_57367", t.uint8_t),
+        }
+    )
+
+    def _update_attribute(self, attrid, value):
+        if attrid == ATTR_CURRENT_POSITION_LIFT_PERCENTAGE:
+            # Invert the percentage value
+            value = 100 - value
+        super()._update_attribute(attrid, value)
+
+    async def command(
+        self, command_id, *args, manufacturer=None, expect_reply=True, tsn=None
+    ):
+        """Override default command to invert percent lift value."""
+        if command_id == CMD_GO_TO_LIFT_PERCENTAGE:
+            percent = args[0]
+            percent = 100 - percent
+            v = (percent,)
+            return await super().command(command_id, *v)
+        return await super().command(
+            command_id,
+            *args,
+            manufacturer=manufacturer,
+            expect_reply=expect_reply,
+            tsn=tsn
+        )
+
+
+class SEDiagnostic(CustomCluster, Diagnostic):
+
+    attributes: dict[int, ZCLAttributeDef] = Diagnostic.attributes.copy()
+
+    # TODO: Check -> this attr don't seems to be manufacturer related (no "manf_id")
+    attributes.update(
+        {
+            0xFFFD: ("unknown_attribute_65533", t.uint16_t),
+        }
+    )
+
+
+class SESpecificCluster(SEManufCluster):
+    cluster_id = 0xFF17
+
+    attributes = {
+        0x0000: ("unknown_attribute_0", t.enum8),
+        0x0001: ("unknown_attribute_1", t.enum8),
+        0x0010: ("unknown_attribute_16", t.uint8_t),
+        0x0011: ("unknown_attribute_17", t.uint16_t),
+        0x0020: ("unknown_attribute_32", t.uint8_t),
+        0x0021: ("unknown_attribute_33", t.uint16_t),
+        0xFFFD: ("unknown_attribute_65533", t.uint16_t),
+    }
+
+
+class SEPiloteMode(int):
+    """
+    Might be useful for :
+    - CCTFR6700 (manufacturerCode seems diff: 0x105e)
```

Comment:
Hi,

I've noticed this code in the PR. I have a CCTFR6700 on Home Assistant (with no Wiser Hub) and I'm looking for a way to configure the pilot mode on the device which seem to be necessary to make the device talk using the "fil pilot". However I don't see this code used anyway and while I tried to use the PR as a custom zha quirks on my HA install, it does not seem to be taken into account by my device. Is this device supposed to be supported and would you have any suggestion on how to allow the configuration of the pilot mode from the HA UI ?
(I've learned about the schneider_pilot_mode field on this page https://www.zigbee2mqtt.io/devices/CCTFR6700.html)

I'm not sure if this is a right place to discuss about the support of this device in the PR, so if there is another place to discuss this do not hesitate to tell me.

Reply:
Done: https://github.com/zigpy/zha-device-handlers/issues/2079